quantumdb-cli/src/main/java/io/quantumdb/cli/commands/Cleanup.java

```diff
@@ -0,0 +1,107 @@
+package io.quantumdb.cli.commands;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static io.quantumdb.core.schema.operations.SchemaOperations.cleanupTables;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import io.quantumdb.cli.utils.CliException;
+import io.quantumdb.cli.utils.CliWriter;
+import io.quantumdb.cli.utils.CliWriter.Context;
+import io.quantumdb.core.backends.Backend;
+import io.quantumdb.core.backends.Config;
+import io.quantumdb.core.backends.DatabaseMigrator.MigrationException;
+import io.quantumdb.core.migration.Migrator;
+import io.quantumdb.core.schema.operations.Operation;
+import io.quantumdb.core.utils.RandomHasher;
+import io.quantumdb.core.versioning.ChangeSet;
+import io.quantumdb.core.versioning.Changelog;
+import io.quantumdb.core.versioning.State;
+import io.quantumdb.core.versioning.Version;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class Cleanup extends Command {
+
+	@Override
+	public Identifier getIdentifier() {
+		return new Identifier("cleanup", "Renames all tables back to their original name.");
+	}
+
+	public void perform(CliWriter writer, List<String> arguments) {
+		try {
+			Config config = Config.load();
+			Backend backend = config.getBackend();
+			writer.write("Scanning database...");
+
+			State state = loadState(backend);
+			Changelog changelog = state.getChangelog();
+
+			checkArgument(state.getRefLog().getVersions().size() == 1, "You may only call this command if you have 1 active version.");
+
+			Version last = changelog.getVersion(state.getRefLog().getVersions().toArray(new Version[0])[0].getId());
+			writer.write(last.getId());
+			ChangeSet changeSet = new ChangeSet("cleanup_" + RandomHasher.generateHash(), "QuantumDB");
+			Operation operation = cleanupTables();
+			changelog.addChangeSet(last, changeSet, operation);
```

Comment:
I implemented it by copying the fork command and adding only this changeset

Reply:
Since you still want to let clients continue using the old version (zero downtime), you cannot remove anything just yet. First, the old version needs to be dropped to remove all triggers, sync functions, and old tables. Then the database administrator can manually remove (or automatically by having another command/parameter of the cleanup command) the quantumdb schema to remove all metadata from the database and removing QuantumDB.